script.js

```diff
@@ -1 +1,189 @@
-//CEOS 19기 프론트엔드 파이팅🔥 ദ്ദി˶ˊᵕˋ˵)
+/*
+  checkboxNumber: todoCard가 unique한 id를 갖기 위한 변수
+*/
+let checkboxNumber = 0;
+
+//todoCard 만들기
+const createTodoCard = (todo, isDone) => {
+  const todoCard = document.createElement('div');
+  const checkbox = document.createElement('input');
+  const todoContent = document.createElement('label');
+  const todoDelete = document.createElement('div');
+
+  todoCard.className = 'todoCard';
+  checkbox.type = 'checkbox';
+  checkbox.className = 'todoCheckbox';
+  checkbox.id = `todoCheckbox${checkboxNumber}`;
+  if (isDone) {
+    checkbox.checked = true;
+    todoCard.className = 'doneCard';
+  } else {
+    todoCard.className = 'todoCard';
+  }
+  todoContent.className = 'todoContent';
+  todoContent.htmlFor = `todoCheckbox${checkboxNumber++}`;
+  todoContent.textContent = todo;
+  todoDelete.className = 'todoDelete';
+  todoDelete.textContent = 'X';
+
+  todoCard.append(checkbox, todoContent, todoDelete);
+  if (isDone) {
+    document.querySelector('.doneSection').append(todoCard);
+  } else {
+    document.querySelector('.todoSection').append(todoCard);
```

Comment:
todoSection이나 doneSection에 ` document.querySelector`를 실행하여 매번 DOM 요소에 접근하고 있는 것 같아서, 이는 성능저하에 큰 영향을 끼친다고 합니다!
```suggestion
const todoSection = document.querySelector('.todoSection');
const doneSection = document.querySelector('.doneSection');
```
이런식으로 DOM요소를 미리 찾아 변수를 저장해서 쓰는 것도 코드의 양과 성능 향상에 도움이 될 수 있을 것 같습니다.